driver/ppm_events_public.h

```diff
@@ -1555,8 +1555,10 @@ enum syscall_flags {
 	UF_USED = (1 << 0),
 	UF_NEVER_DROP = (1 << 1),
 	UF_ALWAYS_DROP = (1 << 2),
-	UF_SIMPLEDRIVER_KEEP = (1 << 3),
+	UF_SIMPLEDRIVER_KEEP = (1 << 3), ///< Mark a syscall to be kept in simpledriver mode, see scap_enable_simpledriver_mode()
 	UF_ATOMIC = (1 << 4), ///< The handler should not block (interrupt context)
+	UF_UNINTERESTING = (1 << 5), ///< Marks a syscall as not interesting. Currently only used by BPF probe to avoid tracing uninteresting syscalls.
```

Comment:
I'm not sure I understand why you want a new flag here. Shouldn't we use existing flags?

Reply:
What flag would you use instead?

Reply:
Oh, I was confusing these flags with the flags in the events table. Never mind.